package.json

```diff
@@ -28,7 +28,7 @@
   ],
   "homepage": "https://github.com/appium/appium-uiautomator2-server",
   "scripts": {
-    "bump-gradle-version": "node ./scripts/gradle-version-update.js --package-version=${npm_package_version} && git add app/build.gradle",
+    "bump-gradle-version": "node ./scripts/gradle-version-update.js --package-version=${npm_package_version} && git add app/build.gradle.kts",
```

Comment:
Should this git add be `gradle.properties` to commit the version update by `gradle-version-update.js`?